extensions/pubsub/standard/sections/clause_7_pubsub.adoc

```diff
@@ -39,18 +39,22 @@ Based on research and testing, the Pub/Sub White Paper recommended the use of As
 }
 ----
 
+NOTE: the media type for an AsyncAPI document may change in the future as decided by the AsyncAPI community.
+
 include::../requirements/pubsub/REQ_rc-api.adoc[]
 
 ==== Providing notification metadata as an OGC API endpoint
 
-For Brokers providing notification metadata (as opposed to actual data payloads), an implementation of OGC API Building Blocks can, in parallel, readily provide GeoJSON-based notification messages via an OGC API - Features endpoint. Providing message payloads via an implementation of OGC API Standard(s) provides the additional benefit of querying for past messages over time in case of a lost connection.
+For Brokers providing notification metadata (as opposed to actual data payloads), an implementation of OGC API Building Blocks can, in parallel, readily provide GeoJSON-based notification messages via an OGC API - Features endpoint. Providing message payloads via an implementation of OGC API Standard(s) provides the additional benefit of querying for past messages over time in case of a lost connection.  See <<pubsub-geojson-fg-json-message-payload-section>> for more information.
 
 ==== Providing Pub/Sub links to collection updates
```

Comment:
My comment in #451 still applies. These links are not web links. The link will appear broken and useless to a client that understands web links (resolving the `href` will not return a resource in the stated media type or that meets the semantics of the link relation type). I think EDR Part 2 should not represent this information in that way and a different approach should be used to represent this information.

If the EDR SWG decides to follow the current approach, at least remove the Features examples. Examples and requirements for Features should be in the scope of the Features API SWG (and I do not see that such an approach would be used for those resources).